### PR TITLE
feat: kickoff implicit account creation flow

### LINKED ISSUE #7811: [Task]: Add implicit account creation router

### PR DESCRIPTION
**Pull request by evavirseda**
## Changelog

```
- Add new account route
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/account/AccountRouter.svelte [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
```
Comment (cpl121) [inline packages/desktop/views/account/AccountRouter.svelte:1]: We would have to add a file `index.js` in `views/account`, then we could add `AccountRouter` to the import of `'@views'` in `packages/desktop/App.svelte`

File: packages/shared/lib/core/router/enums/app-route.enum.ts [hunk 2]
```diff
@@ -2,4 +2,5 @@ export enum AppRoute {
     Dashboard = 'dashboard',
     Login = 'login',
     Onboarding = 'onboarding',
+    Account = 'account',
```
Comment (cpl121) [inline packages/shared/lib/core/router/enums/app-route.enum.ts:5]: I think the name may be more precise. What do you think of `implicitAccountCreation`?

**Review by cpl121 — CHANGES_REQUESTED** [review]
some comments 🙏🏼

File: packages/desktop/App.svelte [hunk 3]
```diff
@@ -41,7 +41,7 @@
     } from '@desktop/routers'
     import { downloadNextNftInQueue, nftDownloadQueue } from '@core/nfts'
     import features from '@features/features'
-    import { OnboardingRouterView } from '@views/onboarding'
+    import { OnboardingRouterView, ImplicitAccountCreationRouterView } from '@views'
```
Comment (cpl121) [inline packages/desktop/App.svelte:44]: Duplicate this import `from '@views'`

**Review by cpl121 — CHANGES_REQUESTED** [review]
NJ!

**Review by cpl121 — APPROVED** [review]
NJ

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
explained what needs to be updated to @cpl121 please discuss with him what needs to be updated
sorry sorry 💔💔 💔

File: packages/desktop/views/dashboard/wallet/Wallet.svelte [hunk 4]
```diff
@@ -1,44 +1,13 @@
 <script lang="ts">
-    import { AssetList, Overflow, Pane, ReceiveAddressButton } from '@ui'
-    import { AccountSummary, AccountActivity, SendButton } from '@components'
-    import { selectedWalletAssets } from '@core/wallet'
-    import { selectedWallet } from '@core/wallet/stores'
-    import features from '@features/features'
+    import { WalletMainView } from './views'
+
+    const hasImplicitAccount: boolean = false
```
Comment (begonaalvarezd) [inline packages/desktop/views/dashboard/wallet/Wallet.svelte:4]:
```suggestion
    const hasAccount: boolean = false
```

File: packages/desktop/views/dashboard/wallet/Wallet.svelte [hunk 5]
```diff
@@ -1,44 +1,13 @@
...
-                </Pane>
-                <Pane>
-                    {#if features?.wallet?.activityHistory?.enabled}
-                        <AccountActivity />
-                    {/if}
-                </Pane>
-            </div>
-        {/key}
-    </wallet-container>
+{#if hasImplicitAccount}
```
Comment (begonaalvarezd) [inline packages/desktop/views/dashboard/wallet/Wallet.svelte:7]:
```suggestion
{#if hasAccount}
```

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**